web/tests/functional/diff_local_remote/test_diff_local_remote.py

```diff
@@ -574,6 +574,38 @@ def test_diff_codeclimate_output(self):
 
         shutil.rmtree(export_dir)
 
+    def test_diff_no_trim_codeclimate_output(self):
+        """ Test codeclimate output when using diff and don't set env vars. """
+        base_run_name = self._run_names[0]
+
+        export_dir = os.path.join(self._local_reports, "export_dir")
```

Comment:
export_dir_name?

Reply:
`_name` can be used to represent a file name and `_path` to represent a file path. For this reason I changed it to `export_dir_path`.